Resources/Controllers/BottomSheet/README.md

```diff
@@ -0,0 +1,159 @@
+#  BottomSheet
+
+A UI component that effectively replaces and extends native UIActionSheet.  
+
+## Overview  
+
+BottomSheet has the following characteristics:
+- dynamically expandable based on the content height
+- swipe down gesture to dismiss
+- tap gesture behind the content to dismiss
```

Comment:
These (the gestures) should be left for the users to implement/integrate in any way they want. The Wizard itself should be as self-contained as possible. It can actually be used as an embedded view controller inside another to provide context for a wizard(multi step)-like UI.